lib/sse_test_server/handler_utils.ex

```diff
@@ -0,0 +1,30 @@
+defmodule SSETestServer.HandlerUtils do
```

Comment:
There's no way to make a module private is there?

Reply:
Nope. A private function is just a function that isn't exported from its module. There isn't really any equivalent grouping abstraction for modules that would make this work cleanly.